docs/guides/nixos/README.md

```diff
@@ -0,0 +1,33 @@
+# Deploy OpenVSCode Server to NixOS
+
+## Prerequisites
+
+None
+
+## Setup
+
+1. Add the following lines to your `/etc/nixos/configuration.nix`:
+
+```nix
+  virtualisation.oci-containers.containers = {
+     openvscode-server = {
+       image = "gitpod/openvscode-server:latest"; # choose between :latest or :nightly
+       ports = ["3000:3000"];
+       volumes = [
+         "/srv:/home/workspace:cached" # change /srv as appropriate
+       ];
+     };
+  };
```

Comment:
Do we need the extra indent at the beginning?

Reply:
Fixed